packages/material-ui/src/Slide/Slide.js

```diff
@@ -11,8 +11,10 @@ import { ownerWindow } from '../utils';
 
 // Translate the node so it can't be seen on the screen.
 // Later, we're going to translate the node back to its original location with `none`.
-function getTranslateValue(direction, node) {
+function getTranslateValue(direction, node, targetRef) {
   const rect = node.getBoundingClientRect();
+  const targetNode = targetRef?.current;
```

Comment:
AFAIK, we can't accept a ref, it needs to be DOM node. Why? Because we have no control over the timing resolution of the ref. The Popper/Popover/useScrollTrigger is not using a ref for this reason. 
```suggestion
  const targetNode = target;
```

Reply:
Do you mean the `anchorEl` in `Popper` and `Popover`? That's a great idea. Maybe I change the `targetRef` prop to `anchorEl` in `Slide` component?

Reply:
`target` as an attribute name wouldn't work because already used by HTML. I'm already confused, why target? It seems to be a container. So +1 for naming it `container` like in the Modal.

Reply:
Yes, `target` is not good enough. But I'm not sure what's the difference between `container` like in the Modal and `anchorEl` like in the popover. They seem almost the same. Which one is better for the `Slide` scenario?